### PR TITLE
Mailroom and Series

### PR DESCRIPTION
**Pull request by Misseswaddles**

### CODE REVIEW

**Comment by Misseswaddles** [comment]
Completed dictionary lab, Mailroom 1 and Series.py

File: students/C_Farris/session02/series.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    assert lucas(7) == 29
+
+    #test sum_series to ensure it returns accurate values for Fib and lucas numbers
+    assert sum_series(0,2,1) == 2
+    assert sum_series(1,2,1) == 1
+    assert sum_series(2,2,1) == 3
+    assert sum_series(3,2,1) == 4
+    assert sum_series(4,2,1) == 7
+    assert sum_series(5,0,1) == 5
+    assert sum_series(6,0,1) == 8
```
Comment (charlesrobison) [inline students/C_Farris/session02/series.py]: Looks good - nice assert statements!

File: students/C_Farris/session03/Mailroom_1.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+#Created by: Carol Farris
+#Purpose: Mailroom 1/2
+#Goal: Create rough framework for Mailroom 
+#Notes: I did jump to using a dictionary to hold the donors, rather than a tuple/list
+
+import sys
+     
+
+#Part 1 of Thank You letter. Gets information from user and stores it.
+def thankyou():
```
Comment (charlesrobison) [inline students/C_Farris/session03/Mailroom_1.py]: Make sure your function names are snake_case: def thank_you():

File: students/C_Farris/session03/Mailroom_1.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    getPerson = ''
```
Comment (charlesrobison) [inline students/C_Farris/session03/Mailroom_1.py]: This should be snake_case too per PEP8: get_person

File: students/C_Farris/session03/Mailroom_1.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    while getPerson not in donor_db :
+        getPerson = input('Please type the donors first and last name or '
+                       'type list to get donor list ==>')
+        getPerson = getPerson.strip().lower().title()
```
Comment (charlesrobison) [inline students/C_Farris/session03/Mailroom_1.py]: Nice use of lower and title here!

File: students/C_Farris/session03/Mailroom_1.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+if __name__ == '__main__' :
+    donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "John Galt": [25.00, 9038.01, 0.01]
+            } 
+
+    main()
```
Comment (charlesrobison) [inline students/C_Farris/session03/Mailroom_1.py]: Works great - nice job!

File: students/C_Farris/session04/Mailroom_2.py [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    user_choices = {'t': thankyou, 'r': makereport, 'x': exitout}
+
+    donor_db = {"William Gates, III": [653772.32, 12.17],
+            "Jeff Bezos": [877.33],
+            "Paul Allen": [663.23, 43.87, 1.32],
+            "Mark Zuckerberg": [1663.23, 4300.87, 10432.0],
+            "John Galt": [25.00, 9038.01, 0.01]
+            } 
+
+    main()
```
Comment (charlesrobison) [inline students/C_Farris/session04/Mailroom_2.py]: Doesn't look like this one's quite finished yet.  I'll go ahead and merge, so please do a new PR when ready for review.

**Pull request merged**